app/controllers/admin/api/accounts_controller.rb

```diff
@@ -57,11 +57,15 @@ def index
   ##~ @parameter_username = {:name => "username", :description => "Username of the account user.", :dataType => "string", :paramType => "query"}
   ##~ @parameter_email = {:name => "email", :description => "Email of the account user.", :dataType => "string", :paramType => "query"}
   ##~ @parameter_id = {:name => "user_id", :description => "ID of the account user.", :dataType => "integer", :paramType => "query"}
+  ##~ @parameter_provider_key = {:name => "buyer_provider_key", :description => "[Master API] Provider key of the account.", :dataType => "string", :paramType => "query"}
```

Comment:
Why not `@parameter_buyer_provider_key`?

Reply:
well, it does not matter that much.. it's just a "variable" name.

Reply:
I'd say be consistent, just in case.